### PR TITLE
Create ar.lua

### PR DESCRIPTION
**Pull request by elvodkaa**

### CODE REVIEW

File: locales/ar.lua [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    blackout = {
+        enabled = 'تم تمكين التعتيم الآن',
+        enabledc = 'تم تمكين التعتيم الآن',
+        disabled = 'تم تعطيل التعتيم الآن',
+        disabledc = 'تم تعطيل التعتيم الآن',
+    },
+    help = {
+        weathercommand = 'تغيير الطقس',
+        weathertype = 'weathertype',
+        availableweather = 'extrasunny, clear, neutral, smog, foggy, overcast, clouds, clearing, rain, thunder, snow, blizzard, snowlight, xmas, halloween',
```
Comment (MonkeyWhisper) [inline locales/ar.lua:41]: We need these translated.

File: locales/ar.lua [hunk 2]
```diff
@@ -0,0 +1,64 @@
+local Translations = {
+    weather = {
+        now_frozen = 'تم تجميد الطقس الآن',
+        now_unfrozen = 'لم يعد الطقس متجمدا',
+        invalid_syntax = '/weather <weathertype> اكتب',
+        invalid_syntaxc = '/weather <weatherType>',
+        updated = 'تم تحديث الطقس',
+        invalid = 'يوجد \nEXTRASUNNY CLEAR NEUTRAL SMOG FOGGY OVERCAST CLOUDS CLEARING RAIN THUNDER SNOW BLIZZARD SNOWLIGHT XMAS HALLOWEEN ',
```
Comment (MonkeyWhisper) [inline locales/ar.lua:8]: Translate please

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]
Please translate

**Comment by elvodkaa** [comment]
Bro i cant bcz this is for commands so

**Pull request closed without merging**